docs/src/main/sphinx/overview/concepts.md

```diff
@@ -81,72 +81,126 @@ for task execution.
 Workers communicate with other workers and Trino coordinators
 using a REST API.
 
-(trino-concept-data-sources)=
-## Data sources
+(trino-concept-client)=
+## Client
+
+Clients allow you to connect to Trino, submit SQL queries, and receive the
+results. Clients can access all configured data sources using
+[catalogs](trino-concept-catalog). Clients are full-featured applications or
+libraries and drivers that allow you to connect to any application supporting
+that driver, or even your own custom application or script.
+
+Clients include command line tools, desktop applications, web-based
+applications, and software-as-a-service solutions with features such as
+interactive SQL query authoring with editors or rich user interfaces for
+graphical query creation, query running and result rendering, visualizations
+with charts and graphs, reporting, and dashboard creation.
+
+[A comprehensive list with more details for each client is available on the
+Trino website](https://trino.io/ecosystem/client).
+
+Clients can process the returned data from Trino and be used for data pipelines
+across catalogs and from other data sources to Trino catalogs.
+
+From a technical perspective, clients only interact with the
+[coordinator](trino-concept-coordinator) of the Trino cluster and use the
+[](/develop/client-protocol).
+
+
+(trino-concept-data-source)=
+## Data source
+
+Trino is a query engine that you can use to query many different data sources.
```

Comment:
different data sources, including

Reply:
I prefer to have shorter separate sentences.